descope/auth/sdk.go

```diff
@@ -96,12 +96,7 @@ type OTP interface {
 	// In case the request cookie can be renewed an automatic renewal is called and returns a new set of cookies to use.
 	// Use the ResponseWriter (optional) to apply the cookies to the response automatically.
 	// returns a list of cookies or an error upon failure.
-	// This is a shortcut for VerifyCodeWithOptions(method, identifier, code, WithResponseOption(w))
 	VerifyCode(method DeliveryMethod, identifier string, code string, w http.ResponseWriter) (*AuthenticationInfo, error)
-	// VerifyCodeWithOptions - used to verify a SignIn/SignUp based on the given identifier either an email or a phone
```

Comment:
...

Reply:
Why? I have the same exact comment above